src/data/matrix/kronecker.lean

```diff
@@ -221,7 +234,9 @@ lemma mul_kronecker_mul [comm_semiring α]
   (A ⬝ B) ⊗ₖ (A' ⬝ B') = (A ⊗ₖ A') ⬝ (B ⊗ₖ B') :=
 kronecker_map_linear_mul_mul (algebra.lmul ℕ α).to_linear_map mul_mul_mul_comm A B A' B'
 
--- insert lemmas specific to `kronecker` below this line
+lemma kronecker_assoc [comm_semiring α] (A : matrix l m α) (B : matrix n p α) (C : matrix q r α) :
```

Comment:
```suggestion
lemma kronecker_assoc [semigroup α] (A : matrix l m α) (B : matrix n p α) (C : matrix q r α) :
```
I don't think you're using the additive structure on `α`

Reply:
Good point!